package.json

```diff
@@ -132,7 +133,7 @@
     "node-sass": "^6.0.1",
     "nodemon": "^2.0.12",
     "pa11y": "^6.0.1",
-    "playwright": "^1.15.0",
+    "playwright": "^1.13.1",
```

Comment:
was this an issue, would dependabot not try to update again?

Reply:
I was trying older versions of the browser. I can update it again, but for now it's probably more important to unblock people